Automattic/jetpack
# AI Assistant: Register ai-assistant-usage-panel beta extension

Part of #33359.

## Proposed changes:

* Register `ai-assistant-usage-panel` as a beta extension so we can build it with small PRs without showing to the public

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Test on Simple, Atomic, and Jetpack sites
* Go to the block editor
* Open the dev console
* With the `beta extensions` disabled, confirm that `Jetpack_Editor_Initial_State.available_blocks['ai-assistant-usage-panel']` is not set:

<img width="500" alt="Screenshot 2023-10-18 at 16 06 51" src="https://github.com/Automattic/jetpack/assets/6760046/b5b43167-04c7-43d4-86da-b5b0df449b01">

* With the `beta extensions` enabled, confirm that `Jetpack_Editor_Initial_State.available_blocks['ai-assistant-usage-panel']` is set to `available: true`:

<img width="500" alt="Screenshot 2023-10-18 at 16 05 58" src="https://github.com/Automattic/jetpack/assets/6760046/a0fd0800-5bf1-45c9-94c3-f61a03580d17">
